test/verify/check-apps

```diff
@@ -19,8 +19,8 @@
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
 import parent
-from testlib import *
```

Comment:
Is that actually a pycodestyle violation? I didn't notice that yesterday. IMHO that's the kind of check that is a bit over the top.. I. e. this commit is fine, but I wouldn't want to enforce it in test-static-code.

Reply:
Actually, no. It's just python best practices stuff, but not error in pycodestyle. So, no worries about future restrictions.